src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -204,23 +204,19 @@ class ProjectTeams extends AsyncView {
     let {orgId, projectId} = this.props.params;
     let access = new Set(this.props.organization.access);
 
-    return (
-      <PanelBody>
-        {this.state.projectTeams.map(team => {
-          return (
-            <TeamRow
-              access={access}
-              key={team.id}
-              orgId={orgId}
-              projectId={projectId}
-              team={team}
-              onRemove={this.handleRemovedTeam.bind(this, team)}
-              teamCount={this.state.projectTeams.length}
-            />
-          );
-        })}
-      </PanelBody>
-    );
+    return this.state.projectTeams.map(team => {
```

Comment:
We should keep `<PanelBody>` (or add it in `renderBody`) so that all the `<Panel>`s are structured the same

Reply:
good catch. i was a little confused by the changes coming from master on this one ty for the heads up!